client/pages/login.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <AuthForm>
+    <InputField class="w-full" field-name="Email" field-type="text" />
+    <InputField class="w-full" field-name="Password" field-type="password" />
+    <a
+      href="/forgot-password"
+      class="self-end text-xs font-semibold text-white underline underline-offset-2"
+    >
+      forgot password?
+    </a>
+    <button
+      class="place-self-center text-l font-bold text-red bg-white w-24 h-8 rounded border border-solid border-red my-5"
+    >
+      Login
+    </button>
+    <p class="place-self-center">
+      don't have an account?
+      <a
```

Comment:
Please use the the `nuxt-link` component for navigating between internal nuxt pages. 
https://nuxtjs.org/docs/features/nuxt-components/#the-nuxtlink-component